packages/create-app/package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "@backstage/create-app",
+  "description": "Create app package for Backstage",
+  "version": "0.1.1-alpha.12",
+  "private": false,
+  "publishConfig": {
+    "access": "public"
+  },
+  "homepage": "https://backstage.io",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/spotify/backstage",
+    "directory": "packages/create-app"
+  },
+  "keywords": [
+    "backstage"
+  ],
+  "license": "Apache-2.0",
+  "main": "src/index.ts",
```

```suggestion
  "main": "dist/index.cjs.js",
```

I think e2e tests break because specifying `src/index.ts` as the entrypoint causes it to be included in the packages version, just like if you included it in the `files` list. That in turn causes the entrypoint script to believe that it's a local development package, i.e. `isLocal` is set to true.

---

Does that mean that I have to put the `builder` part into the package?

---

Nah I think this change should be enough

---

https://github.com/ayshiff/backstage/pull/1 :grin:

---

Thanks @Rugvip 👍